### PR TITLE
removed usage of old nexus repository

### PR DESCRIPTION
**Pull request by arimer**
❗❗ This PR should only be merged after write access to the old Nexus is deactivated.❗❗

In this PR we do:
- remove the publication to the old projects.itemis nexus
- remove the dependency to the old projects.itemis nexus

### CODE REVIEW

**Comment by arimer** [comment]
FYI: @wsafonov , @alexanderpann  The TC build works now, however due to the fact that we removed the support for the old nexus repository, the [publication to that repository is breaking](https://build.mbeddr.com/buildConfiguration/Mbeddr2_Mbeddr_Gradle_platform/330207?expandBuildProblemsSection=true&hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandBuildChangesSection=true) (obviously). 

I am not sure how to continue here:
- either we change the build configuration on TC and remove the `publishMbeddrPlatformPublicationToMavenRepository`. But this will effect builds of all other branches
- Or we keep this PR open until the write access to the old nexus repo is configured.

Any opinions are welcome.

File: build/com.mbeddr/distribution/build.gradle [hunk 1]
```diff
@@ -6,10 +6,10 @@ configurations {
     jreWin
 }
 
-ext.jdkVersion = '8u112b408.4'
+ext.jdkVersion = '11_0_10-b1145.96'
```
[the remark this answers is not in the document]
Comment (arimer) [reply build/com.mbeddr/distribution/build.gradle:9]: Tbh. I did not pay attention. I will update the version as well 👍🏼

**Comment by arimer** [comment]
@wsafonov Done, please have a second look.

**Comment by arimer** [comment]
Looks like mbeddr is trying to fetch those jdk versions from those two repositries: 
- https://github.com/mbeddr/mbeddr.core/blob/master/build.gradle#L17
- https://github.com/mbeddr/mbeddr.core/blob/master/build.gradle#L18

```
ext.defaultSnapshotRepository = 'https://artifacts.itemis.cloud/repository/maven-mps-snapshots/'
ext.defaultReleaseRepository = 'https://artifacts.itemis.cloud/repository/maven-mps-releases/'
```

instead from the publically visible one: https://artifacts.itemis.cloud/repository/maven-mps/

I do not understand the reason for that atm.

**Comment by wsafonov** [comment]
@arimer I have cleaned up the dependency repositories, so that publication repos are not used for dependency resolution. This has resolved the local build issue, since publication repos are not open for anonymous access and require valid credentials for using them for dependency resolution.

The issue on the server was however a different one: for the specified JBR version there was already a Windows artefact `com.jetbrains.jdk:jbr_nomod:11_0_12-b1504.28:windows-x64@tgz` downloaded on all build agents in the corresponding local Maven repo, apparently through some other Maven build on the server. Because we specified `mavenLocal()` repo as the first repo for dependency resolution, Gradle matched that repository when generally looking for the component `com.jetbrains.jdk:jbr_nomod:11_0_12-b1504.28` , but failed to find the other JBR artefact for the OSX platform `com.jetbrains.jdk:jbr_nomod:11_0_12-b1504.28:osx-x64@tgz` for that component, because it was never downloaded before by Maven. And because of that the overall build failed. The solution was removing `mavenLocal()` from the list of dependency repositories. This is also in general recommended for Gradle, see https://docs.gradle.org/current/userguide/declaring_repositories.html#sec:case-for-maven-local for details.

**Review by wsafonov — APPROVED** [review]

**Comment by arimer** [comment]
@wsafonov thanks for taking care 💪🏼

**Pull request merged**